dbcon/execplan/calpontsystemcatalog.cpp

```diff
@@ -1691,16 +1706,13 @@ uint64_t CalpontSystemCatalog::nextAutoIncrValue ( TableName aTableName)
 
 }
 
-int32_t CalpontSystemCatalog::autoColumOid ( TableName aTableName)
+int32_t CalpontSystemCatalog::autoColumOid (TableName aTableName, int lower_case_table_names)
 {
-    transform( aTableName.schema.begin(), aTableName.schema.end(), aTableName.schema.begin(), to_lower() );
-    transform( aTableName.table.begin(), aTableName.table.end(), aTableName.table.begin(), to_lower() );
-
```

Comment:
See above

Reply:
I understand we are not calling autoColumnOid with the 2nd argument. But for completeness, we should add the if block here since aTableName is used later on in this func.

Reply:
I did